01-geodata.qmd

```diff
@@ -12,7 +12,7 @@ geospatial data science, we need to introduce important concepts, which
 will be the foundations of **Part II**, **Part III** and **Part IV**.
 
 In this chapter, we define **geospatial data** and introduce a
-**universal representation** for it that is ideal for geostatistical
+**universal representation** for such data which is ideal for geostatistical
```

Comment:
"for it" isn't correct English? I thought it was clear from the context.

Reply:
Technically, it is ok. I thought it sounded a little weird. The issue is "that". I think a combo of our lines is best:

> **universal representation** for it which is ideal for geostatistical